vizro-core/mkdocs.yml

```diff
@@ -55,7 +55,7 @@ nav:
       - Contribute to Vizro: pages/explanation/contributing.md
       - Documentation style: pages/explanation/documentation-style-guide.md
       - Authors: pages/explanation/authors.md
-  - Examples:
+  - Try out!:
```

Comment:
```suggestion
  - Gallery:
```

Reply:
Can I suggest we use this because the punctuation looks odd, and also it is what we call it.